python/setup.py

```diff
@@ -48,5 +48,6 @@
         path.join(UTILITIES_PATH, "submit_recovery_share.sh"),
         path.join(UTILITIES_PATH, "verify_quote.sh"),
```

Comment:
Should the new utility be included in `scripts` so that it's available in the path when the `ccf` Python package is installed?